aten/src/ATen/native/mkldnn/MKLDNNCommon.cpp

```diff
@@ -67,6 +67,17 @@ ideep::tensor& itensor_from_mkldnn(const MKLDNNTensor& mkldnn_tensor) {
   return mklimpl->unsafe_opaque_handle()->get_target();
 }
 
+ideep::tensor itensor_view_from_dense(const Tensor& tensor) {
+  AT_ASSERTM(
+      tensor.type_id() == CPUTensorId(),
+      "itensor_view_from_dense expects dense CPU tensor input");
```

Comment:
probably want to check dtype is float too if we are checking it's a CPUTensorId.